.github/workflows/phpunit-tests.yml

```diff
@@ -0,0 +1,144 @@
+            # Since the tests are currently run within the Docker containers where the PHP version varies,
+            # the same PHP version needs to be configured for the action runner machine so that the correct
+            # dependency versions are installed and cached.
+            ##
+            - name: Set up PHP
+              uses: shivammathur/setup-php@1a18b2267f80291a81ca1d33e7c851fe09e7dfc4 # v2.22.0
+              with:
+                  php-version: '${{ matrix.php }}'
+                  coverage: none
+
+            # Ensure that Composer installs the correct versions of packages.
+            - name: Override PHP version in composer.json
+              run: composer config platform.php ${{ matrix.php }}
+
+            # The spatie/phpunit-watcher package is not compatible with PHP < 7.2.
+            # It must be removed before running the tests.
+            - name: Remove incompatible Composer packages
+              if: ${{ matrix.php < '7.2' }}
+              run: composer remove spatie/phpunit-watcher --dev --no-update
+
+            # Since Composer dependencies are installed using `composer update` and no lock file is in version control,
+            # passing a custom cache suffix ensures that the cache is flushed at least once per week.
+            - name: Install Composer dependencies
+              uses: ramsey/composer-install@83af392bf5f031813d25e6fe4cd626cdba9a2df6 # v2.2.0
+              with:
+                  custom-cache-suffix: $(/bin/date -u --date='last Mon' "+%F")
+
+            - name: Install npm dependencies
+              run: |
+                  npm ci
+                  npm run build
+
+            - name: Docker debug information
+              run: |
+                  docker -v
+                  docker-compose -v
+
+            - name: General debug information
+              run: |
+                  php --version
+                  npm --version
+                  node --version
+                  curl --version
+                  git --version
+                  svn --version
+                  composer --version
+                  locale -a
+
+            - name: Start Docker environment
+              run: npm run wp-env start
+
+            - name: Log running Docker containers
+              run: docker ps -a
+
+            - name: Docker container debug information
+              run: |
+                  npm run wp-env run tests-mysql "mysql --version"
+                  npm run wp-env run tests-wordpress "php --version"
+                  npm run wp-env run tests-wordpress "php -m"
+                  npm run wp-env run tests-wordpress "php -i"
+                  npm run wp-env run tests-wordpress "/var/www/html/wp-content/plugins/gutenberg/vendor/bin/phpunit --version"
+                  npm run wp-env run tests-wordpress "locale -a"
+
+            - name: Running single site unit tests
+              run: npm run test:unit:php
+              if: ${{ ! matrix.multisite }}
```

Comment:
Nitpick: in most repos, the convention is to have the `if:` above the `run:`

Reply:
Fixed in 9df15b6cb010f0c071ae3da8856dec1d03e519df